examples/remotivelabs-feeder/README.md

```diff
@@ -21,6 +23,9 @@ CREATE DATABASE root.test2
 ```
 CREATE ALIGNED TIMESERIES root.test2.dev1(`Vehicle.Speed` FLOAT, `Vehicle.Chassis.Accelerator.PedalPosition` INT32, `Vehicle.Powertrain.Transmission.CurrentGear` INT32, `Vehicle.Powertrain.TractionBattery.NominalVoltage` INT32, `Vehicle.Powertrain.TractionBattery.StateOfCharge.CurrentEnergy` FLOAT, `Vehicle.Chassis.SteeringWheel.Angle` INT32, `Vehicle.CurrentLocation.Longitude` DOUBLE, `Vehicle.CurrentLocation.Latitude` DOUBLE, `Vehicle.CurrentLocation.HorizontalAccuracy` DOUBLE)
 ```
+### RealmDB schema
```

Comment:
Why is RealmDB mentioned here if the outputs are IoTDB or WS Server (which presumably takes care of the schema)? Are you intending to support direct write to Realm as well?